src/modules/groups/members/MembersList.tsx

```diff
@@ -110,6 +141,10 @@ const MembersList = ({groupId}: IProps) => {
                         >
                             Add Member(s)&nbsp;&nbsp;
                         </Button>
+                        :
+                        null
+                    }
+                    {console.log(profile)}
```

Comment:
I think you forgot to remove the `console.log()`

Reply:
Oh thank you